src/libs/constants/constants.ts

```diff
@@ -1,3 +1,7 @@
 export { APPLICATION_NAME } from "./application-name.constant.js";
 export { FIRST_ARRAY_ELEMENT } from "./first-array-element.constant.js";
 export { LAST_ARRAY_ELEMENT } from "./last-array-element.constant.js";
+export {
+	NUMBER_OF_REVIEWS_TO_RENDER,
+	REVIEWS,
+} from "./reviews-array.constant.js";
```

Comment:
one constant per file, filename should match the item exported, change it to just `reviews.constant.js`